test/unit/calculators/benefit_cap_calculator_configuration_test.rb

```diff
@@ -112,6 +120,15 @@ class BenefitCapCalculatorConfigurationTest < ActiveSupport::TestCase
             assert_equal "other_first_question", @config.benefits(:other).fetch("first_benefit")["question"]
           end
         end
+        context "merge questions" do
+          should "get all benefit questions from multiple configuration sets" do
+            questions = @config.all_questions
+            assert_includes questions.values, "other_first_question"
+            assert_includes questions.values, "other_second_question"
+            assert_includes questions.values, "third_question"
+            assert_includes questions.values, "other_third_question"
```

Comment:
I find it strange that we're not checking that `first_question` and `second_question` are included here, but we are checking all the others.